.travis.yml

```diff
@@ -4,8 +4,6 @@ sudo: false
 
 matrix:
   include:
-    - python: 2.7
-      env: TOXENV=py27
     - python: 3.4
       env: TOXENV=py34
     - python: 3.5
```

Comment:
@mmerickel should we drop 3.4 and 3.5, too, since they are EOLed?

Reply:
Yes we should, but we can do that after this is merged. As we will also want to move this project to Github Actions instead (following our established model).